engine/engine/src/engine.cpp

```diff
@@ -943,7 +943,8 @@ namespace dmEngine
         }
 
         int32_t liveupdate_enable = dmConfigFile::GetInt(engine->m_Config, "liveupdate.enabled", 1);
-        if (liveupdate_enable)
+        int32_t liveupdate_mount_on_start = dmConfigFile::GetInt(engine->m_Config, "liveupdate.mount_on_start", 1);
+        if (liveupdate_enable && liveupdate_mount_on_start)
```

Comment:
we already have this flag in resource system, but `liveupdate.enabled` used in another place as well, so we need new flag for this particular feature

Reply:
Since this flag now explicitly takes on a different meaning, I suggest renaming it as well.

E.g. RESOURCE_FACTORY_FLAGS_NO_LIVE_UPDATE_MOUNT_ON_START
or something of the sorts.

Reply:
I should be "positive" because it's on by default - means we will mount archive if it's on 
So I pick RESOURCE_FACTORY_FLAGS_LIVE_UPDATE_MOUNTS_ON_START to make it clear